gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageIntegrationTest.java

```diff
@@ -61,13 +56,8 @@ private static GoogleCloudStorage getGoogleCloudStorage() throws IOException {
 
   private static GoogleCloudStorage getGoogleCloudStorage(
       GoogleCloudStorageOptions.Builder optionsBuilder) throws IOException {
-    Credential credential = GoogleCloudStorageTestHelper.getCredential();
-    return new ThrottledGoogleCloudStorage(
-        // Allow 2 create or delete bucket operation every second. This will hit rate limits,
-        // but GCS now has back-offs implemented for bucket operations.
-        RateLimiter.create(2),
-        new GoogleCloudStorageImpl(optionsBuilder.build(), credential),
-        EnumSet.of(StorageOperation.DELETE_BUCKETS, StorageOperation.CREATE_BUCKET));
+    return new GoogleCloudStorageImpl(
+        optionsBuilder.build(), GoogleCloudStorageTestHelper.getCredential());
```

Comment:
It would be helpful to include a commit/review comment about why we're removing the rate limiting.

Reply:
Done.